### DISCUSSION TITLE
Can it use a TTS other than the system one?

### DISCUSSION THREAD

**Discussion by qgustavor** [organicmaps/organicmaps#9024] [Q&A] [answered]
At the moment my phone is set to use a broken TTS in order for the phone app, which uses the system TTS, not alert call recordings which, AFAIK, is legal in my area but whoever developed the phone app didn't consider this, and it's great to have as once someone threatened someone close to me and it will be handy in the case it happens again.

I have another TTS service on my phone, is it possible to set up Organic Maps to use it instead?

**Comment by biodranik** [comment, marked as answer]
It is possible in theory, but in practice using something else except a system default is an extremely rare case...
